.github/workflows/test.yml

```diff
@@ -0,0 +1,18 @@
+name: Test
```

Comment:
Could you add the build status badge in the README and delete the travis.yml file?

Reply:
Updated